packages/terra-framework-docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Framework Docs
+
+[![NPM version](https://badgen.net/npm/v/@cerner/terra-framework-docs)](https://www.npmjs.org/package/@cerner/terra-framework-docs)
+[![Build Status](https://badgen.net/travis/cerner/terra-framework)](https://travis-ci.com/cerner/terra-framework)
+
+Contains documentation for packages in the terra-framework monorepo
+
+## Versioning
```

Comment:
Can this section be removed in favor of the mono-repo's versioning doc?
https://github.com/cerner/terra-framework#--versioning

Reply:
I don't think it super matters, but the main place this readme will be seen is on npm, not really associated with the monorepo.
Like this one for terra-toolkit-docs. https://www.npmjs.com/package/@cerner/terra-toolkit-docs

I don't know how much it matters in those readmes to call out semver, but it does fill up space 🤷‍♂️ 

I'd be ok just removing this too.